dg-example-default/src/main/resources/samplemachine.xml

```diff
@@ -71,6 +73,9 @@
             <!-- Example of using '%number' from equivalence class generation  -->
             <assign name="var_2_6" expr="%number(5,2)"/>
 
+            <!-- Example of using '$var_name' from equivalence class generation  -->        <!-- Added by Shraddha Patel -->
+            <assign name="var_2_7" expr="$var_2_2"/>
```

Comment:
My only comment is that we need a high level strategy to figure out the special notation such as '$' and '%' within a variable definition needs supporting escape characters if the user indeed intends to have a $ or % in the variable as a static string.